elements/pl-file-upload/pl-file-upload.js

```diff
@@ -11,6 +11,11 @@ window.PLFileUpload = function(uuid, options) {
     this.uuid = uuid;
     this.files = options.files || [];
     this.acceptedFiles = options.acceptedFiles || [];
+    this.acceptedFilesLowerCase = [];
+    var i;
+    for (i = 0; i < this.acceptedFiles.length; i++) {
```

Comment:
```suggestion
    for (let i = 0; i < this.acceptedFiles.length; i++) {
```

Reply:
I'm fine with this but I'd thought maybe it was written in a style for pre-ES2015 compatibility.